User doc update runai2.23 (#22)
Sydney-Informatics-Hub/dgx-cluster-onboarding-guide: notebooks/jupyter_tutorial.md

Generally, the minimum requirements you need before creating the workload include:

* Being granted permission to an [active project](runai_features.md#runai-projects)
* An [environment](runai_features.md#runai-environments) to run such job
* Have a [data source](runai_features.md#runai-data-sources), e.g. a PVC, to store your input and output data
* Understand the [compute resources](runai_features.md#runai-compute-resources) requirements you need to run the job

In this tutorial, we will create a basic Jupyter Lab workload that allows you to run Jupyter notebooks interactively on the SIH GPU cluster.

## Step 1: Create a workload
Navigate to the "Workload manager" section, select "Workloads", and click on the "NEW WORKLOAD" button. Select "Workspace" from the dropdown menu.

![Create a new workload](../fig/workload_create_workspace.png)

## Step 2: Configure the workload from scratch
Define the necessary information for your workload:

> 

Reviewer: Maybe add a sentence to say Cluster will be set automatically? 

`* The "Cluster" section will be set automatically, you do not need to change this`

Author: added in 6728ca35f611a5b39a41b8637f941b2c9f471122

* Under "Projects" select the project it will be linked to
* Under "Templates" select "Start from scratch" (*i.e.* do not use any existing template)
* Provide a descriptive name for the workload

    ![Project and Template](../fig/workload_definition.png)

> * Select the `jupyter-notebook` environment to create the workload. This environment pulls a [pre-built image](https://hub.docker.com/r/sydneyinformaticshub/dgx-interactive-jupyterlab) (`sydneyinformatics/dgx-interactive-jupyterlab`) that has Jupyter Lab and commonly used data science/AI packages installed, including Tensorflow, pytorch, pandas, and ollama. The `jupyter-notebook` environment is also configured to intialise the Jupyter Lab server directly from the PVC.

Reviewer: ```suggestion
* Under "Environment", select the `jupyter-notebook` environment to create the workload. This environment pulls a [pre-built image](https://hub.docker.com/r/sydneyinformaticshub/dgx-interactive-jupyterlab) (`sydneyinformatics/dgx-interactive-jupyterlab`) that has Jupyter Lab and commonly used data science/AI packages installed, including Tensorflow, pytorch, pandas, and ollama. The `jupyter-notebook` environment is also configured to intialise the Jupyter Lab server directly from the PVC.
```

    ![Software environment](../fig/workload_environment.png)

> * Select the amount of compute resources to run the workload. In this tutorial, we will select the `small-fraction` option that requires 1 H200 GPU with 10% of its memory (~14GB). Depending on the actual workload you're running, this option can be adjusted accordingly.

Reviewer: ```suggestion
* Under "Compute resources", select the amount of compute resources to run the workload. In this tutorial, we will select the `small-fraction` option that requires 1 H200 GPU with 10% of its memory (~14GB). Depending on the actual workload you're running, this option can be adjusted accordingly.
```

    ![Compute resource](../fig/workload_comp_resource.png)

> * Configure the data source to be mounted to the container. Here we select the default PVC created for the project. The mount path inside the container is set to `/scratch/<runai-project-name>`.

Reviewer: ```suggestion
* Click "Data & sources" to expand this section and configure the data source to be mounted to the container. Here we select the default PVC created for the project. The mount path inside the container is set to `/scratch/<runai-project-name>`.
```

>     ![Data resource](../fig/workload_datasource.png)

Reviewer: <img width="3014" height="1732" alt="Image" src="https://github.com/user-attachments/assets/f0db0390-02d2-4d6c-8fc2-a53fd3bb6b29" />

Add an arrow and step 3. to click the Load buttom in the botton right corner

> * Lastly, Click on "CREATE WORKLOAD" to submit the workload to the cluster.

Reviewer: ```suggestion
* Finally, click on "CREATE WORSPACE" to submit the workload to the cluster. The workspace can take a few minutes to initialise.
```

![Connect to the Jupyter Lab interface](../fig/workload_connect_jupyter.png)

## (Optional) Step 4: Inspect system logs
You can also further view details of the workload including

* Event history: This information is especially useful for debugging issues when a workload fails to start or is pending for a long time.
* Metrics: This contains real time GPU and CPU usage, which helps optimise resource utilisation.
* Logs: This provides an easy access to the container's outputs and any error messages.
* Details: This tab includes all container runtime settings.

> ![Workload logs](../fig/workload_logs.png)

Reviewer: <img width="1112" height="488" alt="Image" src="https://github.com/user-attachments/assets/0730623e-cd6b-4ee5-b866-3b755e5ce16b" />

Recommend changing colour for this arrow as it can be hard for some people to see red over green. Maybe a dark blue arrow or circle over the icon?

Author: Good suggestion. New commit to update the figures: 467a2ef15aa45c8a932213d6eadab0d37b4658f1
